kairon/shared/data/data_models.py

```
@@ -0,0 +1,1341 @@
+        except AppException as e:
+            raise ValueError(str(e))
+        return v
+
+    @validator("password")
+    def validate_password(cls, v, values, **kwargs):
+        from kairon.shared.utils import Utility
+
+        try:
+            Utility.valid_password(v.get_secret_value())
+        except AppException as e:
+            raise ValueError(str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)